### ISSUE TITLE
Random tests not passing, as a result of unresolved merge conflicts

### ISSUE THREAD

**Issue by eileenchoe** [dondi/GRNsight#491]
The following 5 tests are not passing as a unintended result of merge conflicts that @mihirsamdarshi and @eileenchoe were sorting out this morning with @NAnguiano 's pull request. Bookmarking them here so we can return to addressing them separately. They are currently commented out while we are focusing on other issues for the rest of the day. 
- errors-adjacency-matrix-modifications extra-column-end-of-sheet should return warnings count error
- errors-adjacency-matrix-modifications extra-row-end-of-sheet should return errors count error
- errors-adjacency-matrix-modifications extra-row-end-of-sheet-output should return errors count error
- warnings-adjacency-matrix-modifications missing-column-end no warnings
- warnings-adjacency-matrix-modifications sheet-named-incorrectly should return a wrong name warning

**Comment by eileenchoe** [comment]
Another adjacency matrix test issue brought up by the merge. Commented out with intention to fix along with this issue
- value-replaced-with-spaces

**Comment by kdahlquist** [comment]
Elevating this now that the zoom work is finished.

**Comment by eileenchoe** [comment]
Fixed the broken tests. The tests were failing because of a few random values that got changed from the merge. After cross checking with the last working beta pre-merge it seems that only those tests listed above were affected.

**Comment by kdahlquist** [comment]
This hasn't been pulled in via a PR yet, though, right?

**Comment by eileenchoe** [comment]
Working on the PR now, had merge conflicts I'm trying to fix.

**Comment by eileenchoe** [comment]
Live on beta 2.1.8

**Comment by kdahlquist** [comment]
I'm going to assign this review to @dondi.

**Comment by dondi** [comment]
I looked at these errors and they look mostly right except for a redundant test that I found in both _errors-adjacency-matrix-modifications.js_ and _warnings-adjacency-matrix-modifications.js_ for `missing-column-end`. Because the test pertained to warnings, I removed it from the _errors_ file and kept just the one in the _warnings_ file. This was committed and pushed directly to `beta`.

The change reduces the test count from 207 to 206, but that is as it should be because it was the same test being performed twice.

Review completed; removing `review requested` tag.
